src/components/input/select.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import DropDown from 'react-native-paper-dropdown';
+import { AppColor } from 'src/common/enums';
+
+import styles from './styles';
+import theme from './theme';
+
+type Props = Omit<
+  React.ComponentPropsWithoutRef<typeof DropDown>,
+  'visible' | 'onDismiss' | 'showDropDown'
+>;
+
+const Select: React.FC<Props> = dropDownProps => {
+  const [open, setOpen] = useState<boolean>(false);
+
+  const show = () => setOpen(true);
+  const hide = () => setOpen(false);
+
+  return (
+    <DropDown
+      mode="flat"
+      activeColor={AppColor.ACCENT}
+      visible={open}
+      showDropDown={show}
+      onDismiss={hide}
+      // eslint-ignore-next-line @typescript-eslint/no-explicit-any
+      theme={theme as any}
```

Comment:
```suggestion
      theme={theme}
```

Reply:
fixed in [e4ed6df](https://github.com/BinaryStudioAcademy/bsa-winter-2022-growup-mobile/pull/16/commits/e4ed6df525c133157c5109fd79f7db8fbaeb773b)